react/tic-tac-toe/src/app/index.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 
 import Game from './screens/Game';
+import Signup from './screens/Signup';
 
 import '../scss/application.scss';
```

Comment:
Inside your application.scss you should have your all your generic styles imports. Like, colors, texts, etc.

Reply:
There is no need to add color and text here 👀 like I said above you should import all of this in your application.scss file @jtorreguitar